client/src/components/TagField.scss

```
@@ -108,3 +108,7 @@
     color: #0071e6;
   }
 }
+
+.ss-tag-field__menu {
+  z-index: 10;
```

```suggestion
  // Prevents the dropdown from hiding behind TinyMCE
  // see https://github.com/JedWatson/react-select/issues/5329
  z-index: 10;
```
Please add this so there's some immediate feedback about why this is needed - it's pretty weird to need a z-index like this so we're bound to want to know about it in the future.